Resources/public/templates/fields/edit/selection.hbt

```diff
@@ -9,6 +8,11 @@
         </label>
     </div>
     <div class="pure-u ez-editfield-input-area ez-default-error-style">
+        <div class="ez-fielddescription-tooltip ez-tail-down-tooltip">
```

Comment:
also I would be nice to use the same HTML element(s) to display the description.